src/WebJobs.Extensions.DurableTask/LifeCycleNotificationHelper.cs

```diff
@@ -272,5 +314,76 @@ private class EventGridEvent
             [JsonProperty(PropertyName = "dataVersion")]
             public string DataVersion { get; set; }
         }
+
+        internal class HttpRetryMessageHandler : DelegatingHandler
+        {
+            private readonly int maxRetryCount;
+            private readonly Func<int, TimeSpan> retryWaitSpanFunc;
+            private int[] retryTargetStatus;
+
+            public HttpRetryMessageHandler(HttpMessageHandler messageHandler, int maxRetryCount, Func<int, TimeSpan> retryWaitSpanFunc, int[] retryTargetStatusCode)
```

Comment:
It looks like all uses of `Func<int, TimeSpan> retryWaitSpanFunc` specify a constant timeout. The public properties require a static number, so I think we should make this a simple `TimeSpan` parameter instead of a `Func`.